simul/runfiles/mininet_cosi.toml

```diff
@@ -0,0 +1,12 @@
+Simulation = "CoSimul"
+Servers = 16
+Bf = 8
+Rounds = 10
+CloseWait = 6000
```

Comment:
You renamed this [above](https://github.com/dedis/cothority/pull/640/files#diff-7c46770087ffa7cc7eefbe702ec35654L84). Either change it back to the original name in the golang code (`CloseWait`) or consistently rename it in all toml files.